tests/fdb/test_populate_fdb.py

```diff
@@ -0,0 +1,126 @@
+import json
+import logging
+import pytest
+
+from common.platform.ssh_utils import prepare_testbed_ssh_keys as prepareTestbedSshKeys
+from ptf_runner import ptf_runner
+
+logger = logging.getLogger(__name__)
+
+# Globals
+PTFRUNNER_QLEN = 1000
+VXLAN_CONFIG_FILE = '/tmp/vxlan_config.json'
```

Comment:
Can you change this file name to a proper file name?

Reply:
done. used vlan_config